hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueContainer.java

```diff
@@ -64,8 +64,8 @@
     private static final int ID_PROMOTION_OFFSET = 100000;
     private final Map<Long, TxQueueItem> txMap = new HashMap<Long, TxQueueItem>();
     private final Map<Long, Data> dataMap = new HashMap<Long, Data>();
-    private final QueueWaitNotifyKey pollWaitNotifyKey;
-    private final QueueWaitNotifyKey offerWaitNotifyKey;
+    private QueueWaitNotifyKey pollWaitNotifyKey;
```

Comment:
Ah, forget it. I didn't see the variable, just the "poll" and "offer". Sure, it's done properly your way.